tests/components/spreadsheet.test.ts

```diff
@@ -399,6 +434,216 @@ describe("Composer interactions", () => {
     fixture.remove();
     app.destroy();
   });
+
+  test("Selecting a range should not scroll the viewport to the current Grid selection", async () => {
```

Comment:
what do you think about moving thoses composer/spreadsheet tests in their own file?

Reply:
I was thinking the same thing yes, I didn't do it because I fear that we start multiplying the test files too much but I really think it's worth in this case.